smooth/components/component.py

```diff
@@ -29,6 +29,11 @@ def __init__(self):
         self.opex = dict()
         self.capex = dict()
 
+        # Emissions values for consumption and installation in [kg/Wh]

```

Comment:
[kg/???]

Reply:
alright kg/Wh would be just an example, I'll change it

maybe kg/* would look nicer (like a regular-expression). As ??? seems like a ToDo

Reply:
https://github.com/rl-institut/smooth/pull/15/commits/66a8875ef6b6af54eb7bd4e9f044f63510cf8295